app/models/student.rb

```diff
@@ -9,7 +9,7 @@ class Student < ApplicationRecord
   has_many :assessments
 
   has_many :enrollments
-  has_many :coordinators, through: :enrollments
```

Comment:
It's like... the student registers at an affiliate and then the coordinator is in charge of that affiliate. The student doesn't register at or with a coordinator... The coordinator is coordinating the affiliate.